probe-rs/src/architecture/riscv/dtm/dtm_access.rs

```diff
@@ -0,0 +1,102 @@
+use crate::architecture::riscv::communication_interface::RiscvError;
+use crate::probe::{CommandResult, DeferredResultIndex};
+use crate::{DebugProbe, DebugProbeError, Probe};
+use bitfield::bitfield;
+use std::fmt;
+use std::time::Duration;
+
+pub trait DtmAccess: Send + fmt::Debug {
+    /// Asserts a reset of the physical pins
+    /// Returns true if the standard assert reset sequence should happen (when they don't exist)
```

Comment:
What do you mean by `when they don't exist`? I'm not sure I understand what the return value does here.

Reply:
I have removed the boolean as there is no way for us to tell if the user if they even have the nRST pin hooked up.